### PR TITLE
enable additional flags for tank test models

### PR DESCRIPTION
**Pull request by aldesilv**
enabled im2col and winograd flags

### CODE REVIEW

**Comment by monorimet** [comment]
This is great, thank you -- I'm doing a rebase that should fix the CI issue encountered here previously.

**Review by monorimet — APPROVED** [review]
LGTM!

**Comment by powderluv** [comment]
can we rebase / land this ?

File: tank/all_models.csv [hunk 1]
```diff
@@ -22,15 +22,15 @@ facebook/deit-small-distilled-patch16-224,linalg,torch,1e-2,1e-3,default,nhcw-nh
...
 nvidia/mit-b0,linalg,torch,1e-2,1e-3,default,None,True,True,True,"https://github.com/nod-ai/SHARK/issues/343"
-resnet101,linalg,torch,1e-2,1e-3,default,nhcw-nhwc,False,False,True,"Vulkan Numerical Error (mostly conv)"
+resnet101,linalg,torch,1e-2,1e-3,default,nhcw-nhwc/img2col,False,False,True,"Vulkan Numerical Error (mostly conv)"
 resnet18,linalg,torch,1e-2,1e-3,default,None,True,True,True,""
 resnet50,linalg,torch,1e-2,1e-3,default,nhcw-nhwc,False,False,True,"Vulkan Numerical Error (mostly conv)"
-resnet50_fp16,linalg,torch,1e-2,1e-2,default,nhcw-nhwc,True,False,True,""
-squeezenet1_0,linalg,torch,1e-2,1e-3,default,nhcw-nhwc,False,False,True,"https://github.com/nod-ai/SHARK/issues/388"
-wide_resnet50_2,linalg,torch,1e-2,1e-3,default,nhcw-nhwc,False,False,True,"Vulkan Numerical Error (mostly conv)"
+resnet50_fp16,linalg,torch,1e-2,1e-2,default,nhcw-nhwc/img2col,True,False,True,""
+squeezenet1_0,linalg,torch,1e-2,1e-3,default,nhcw-nhwc/img2col,False,False,True,"https://github.com/nod-ai/SHARK/issues/388"
```
Comment (monorimet) [inline tank/all_models.csv:33]: We should remove img2col here if it's currently broken (see CI) and merge :+1:

**Review by monorimet — CHANGES_REQUESTED** [review]
See comments

**Comment by powderluv** [comment]
can we land this

**Pull request merged**